pages/_app.tsx

```diff
@@ -1,24 +1,22 @@
-import "react-app-polyfill/ie9";
+import "react-app-polyfill/stable";
 import type { AppProps } from "next/app";
 import React from "react";
 
 import { appWithTranslation } from "next-i18next";
 import { Provider } from "next-auth/client";
 import Base from "../components/globals/Base";
 import "../styles/app.scss";
-import { useHMR } from "../lib/hooks/useHMR";
 import i18nextConfig from "../next-i18next.config";
 
 const SafeHydrate = ({ children }: { children: React.ReactNode }) => {
   return (
-    <div suppressHydrationWarning={Boolean(process.env.CYPRESS)}>
-      {typeof window === "undefined" && process.env.CYPRESS ? null : children}
+    <div suppressHydrationWarning={Boolean(process.env.ISOLATED_INSTANCE)}>
```

Comment:
Not sure what this is doing here. I know it existed before and just changing the variable name but would be helpful to include a comment

Reply:
I'll add a comment in the code.
Essentially this block disables SSR when in testing mode.  This is because in Cypress we're manipulating and mocking the API response calls and the SSR pages were not matching the React rendered pages generating a warning in the browser console.